src/eckit/stats/Statistics.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * (C) Copyright 1996- ECMWF.
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * In applying this licence, ECMWF does not waive the privileges and immunities
+ * granted to it by virtue of its status as an intergovernmental organisation nor
+ * does it submit to any jurisdiction.
+ */
+
+
+#pragma once
+
+#include <iosfwd>
+#include <string>
+
+
+namespace eckit {
+class Parametrisation;
+namespace stats {
+class Field;
+}
+}  // namespace eckit
+
+
+namespace eckit::stats {
+
+
+class Statistics {
+public:
+    // -- Constructors
+
+    Statistics(const Parametrisation&);
+    Statistics(const Statistics&) = delete;
+
+    // -- Destructor
+
+    virtual ~Statistics();
+
+    // -- Operators
+
+    void operator=(const Statistics&) = delete;
```

Comment:
This is curious. I would expect some method to get the statistics _out_...

Reply:
Suggests that the API has different uses than read trivially - and probably needs some documentation commentary at the top.